waffle_configs/bedrock-prod.env

```diff
@@ -79,6 +79,6 @@ SWITCH_VPN_EVENTS_BANNER=off
 SWITCH_VPN_HOLIDAYS_NA_TEST=off
 SWITCH_POCKET_BEST_OF_2023_BANNER=off
 SWITCH_MONITOR_PLUS_BANNER=off
-SWITCH_MOZILLA_MONITOR_BRAND_NAME=on
+SWITCH_MOZILLA_MONITOR_BRAND_NAME=off
```

Comment:
1. it should be switched off only after https://github.com/mozilla/bedrock/pull/14493 lands (not to revert to obsolete branding everywhere for now)
2. at that time the switch can be removed entirely instead?

Reply:
Good call out! 
I spoke w/ the team, and we agreed since this switch is no longer needed, then removing it from the repo is a good idea. Thanks!

Reply:
Actually I should have used "reaches production deployment" instead of "lands" — because what happens is https://whatsdeployed.io/s/RuO/mozilla/bedrock still doesn't have the removals in prod, but the switches are:
- dev: "Mozilla Monitor"
  - https://www-dev.allizom.org/en-GB/firefox/privacy/products/
- prod: "Firefox Monitor"
  - https://www.mozilla.org/en-GB/firefox/privacy/products/

@reemhamz Maybe you can arrange a bedrock tag+release today to fix this? 🙏 /cc @alexgibson